pagerules.go

```diff
@@ -104,18 +104,20 @@ type PageRulesResponse struct {
 // CreatePageRule creates a new Page Rule for a zone.
 //
 // API reference: https://api.cloudflare.com/#page-rules-for-a-zone-create-a-page-rule
-func (api *API) CreatePageRule(zoneID string, rule PageRule) error {
+func (api *API) CreatePageRule(zoneID string, rule PageRule) (*PageRuleDetailResponse, error) {
 	uri := "/zones/" + zoneID + "/pagerules"
 	res, err := api.makeRequest("POST", uri, rule)
 	if err != nil {
-		return errors.Wrap(err, errMakeRequestError)
+		return nil, errors.Wrap(err, errMakeRequestError)
 	}
-	var r PageRuleDetailResponse
-	err = json.Unmarshal(res, &r)
+
+	var pageRuleResp *PageRuleDetailResponse
+	err = json.Unmarshal(res, &pageRuleResp)
```

Comment:
@OJFord Is this taking the address of a pointer? Could probably just pass in `pageRuleResp` without `&` since `pageRuleResp` is already a pointer, no?